charts/astronomer/templates/registry/registry-networkpolicy.yaml

```diff
@@ -71,6 +71,16 @@ spec:
           tier: astronomer
           component: houston-worker
           release: {{ .Release.Name }}
+    - podSelector:
+      matchLabels:
+        tier: astronomer
+        release: {{ .Release.Name }}
+        component: houston-populate-daily-task-metrics
```

Comment:
```suggestion
        matchLabels:
          tier: astronomer
          release: {{ .Release.Name }}
        component: houston-populate-daily-task-metrics
```

Reply:
You know what? If that slipped through unit tests, it means we need unit tests to validate that this file is valid.

Reply:
I'm going to add tests in a different branch so we can unblock this, and because it's becoming quite messy.